commands/room_exit.py

```diff
@@ -32,8 +34,7 @@ class CmdTerrain(MuxCommand):
     Notes:
       If <room> is omitted, defaults to your current location.
 
-      Acceptable values for <terrain> are EASY, MODERATE, DIFFICULT,
-      MUD, ICE, QUICKSAND, SNOW, VEGETATION, THICKET, or DEEPWATER.
+      Acceptable values for <terrain> can be found in typeclasses/rooms.py.
```

Comment:
You shouldn't take out the acceptable terrain message - that's actually for the command help, we shouldn't have to read the source to know. :]